integration-tests/bouncycastle-jsse/src/test/java/io/quarkus/it/bouncycastle/BouncyCastleJsseTestCase.java

```diff
@@ -46,13 +45,15 @@ protected void doTestListProviders() {
                 .setKeyStore("client-keystore.jks", "password")
                 .setTrustStore("client-truststore.jks", "password")
                 .build();
-        RestAssured.given()
+        String response = RestAssured.given()
                 .spec(spec)
                 .when()
                 .get("/jsse/listProviders")
                 .then()
                 .statusCode(200)
-                .body(containsString("BC,BCJSSE,SunJSSE"));
+                .extract().asString();
+        assertTrue(response.startsWith("Identity: CN=client"));
+        assertTrue(response.contains("BC,BCJSSE,SunJSSE"));
```

Comment:
I think you can include these assertions in the `body(...)` statement above?

Reply:
@gastaldi Give me a few secs, consider it done :-)

Reply:
Updated it